_posts/en/newsletters/2025-01-17-newsletter.md

```diff
@@ -0,0 +1,153 @@
+---
+title: 'Bitcoin Optech Newsletter #337'
+permalink: /en/newsletters/2025/01/17/
+name: 2025-01-17-newsletter
+slug: 2025-01-17-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes continued discussion about rewarding
+pool miners with tradeable ecash shares and describes a new proposal for
+enabling offchain resolution of DLCs.  Also included are our regular
+sections announcing new releases and release candidates and describing
+notable changes to popular Bitcoin infrastructure software.
+
+## News
+
+- **Continued discussion about rewarding pool miners with tradeable ecash shares:**
+  [discussion][ecash tides] continued since our [previous
+  summary][news304 ecashtides] of a Delving Bitcoin thread about paying
+  [pool miners][topic pooled mining] with ecash for each share they
+  submitted.  Previously, Matt Corallo [asked][corallo whyecash] why a
+  pool would implement the extra code and accounting to handle tradable ecash
+  shares when they could simply pay miners using a normal ecash mint (or
+  via LN).  David Caseria [replied][caseria pplns] that in some _pay per
+  last N shares_ ([PPLNS][topic pplns]) schemes, such as
+  [TIDES][recap291 tides], a miner might need to wait for the pool to
+  find several blocks, which might take days or weeks for a small pool.
+  Instead of waiting, a miner with ecash shares could immediately sell
+  them on an open market (without disclosing to the pool or any third
+  party anything about their identity, not even any ephemeral identity
+  they used when mining).
+
+  Caseria also noted that existing mining pools find it financially
+  challenging to support the _full paid per share_ ([FPPS][topic fpps])
+  scheme where a miner is paid proportional to the entire block reward
+  (subsidy plus transaction fees) when they create a share.  He didn't
+  elaborate, but we understand the problem to be variance in fees
+  forcing pools to keep large reserves.  For example, if a pool miner
+  controls 1% of hashrate and creates shares on a template with about
+  1,000 BTC in fees and 3 BTC in subsidy, they would be owed by their
+  pool about 10 BTC.  However, if the pool does mine that block and,
+  when they do mine a block, fees are back down to a fraction of the
+  block reward, the pool might only have 3 BTC total to split between
+  all of its miners, forcing it to pay from its reserves.  If that
+  happens too many times, the pool's reserves will be exhausted and it
+  will go out of business.  Pools address this in various ways,
+  including using various [proxies for actual fees][news304 fpps proxy].
```

Comment:
```suggestion
  including using [proxies for actual fees][news304 fpps proxy].
```
Probably don't want to use "various" twice so close together in the same sentence; I don't think the second one is needed.